rlclientlib/vw_model/safe_vw.cc

```diff
@@ -192,7 +245,9 @@ bool safe_vw::is_compatible(const std::string& args) const {
   //TODO: proper implentation for more than 2 algorithms. Changes inside VW are required
   const bool is_ccb_init = args.find("ccb_explore_adf") != std::string::npos;
   const bool is_ccb = _vw->options->was_supplied("ccb_explore_adf");
-  return is_ccb_init == is_ccb;
+  const bool is_slates_init = args.find("slates") != std::string::npos;
+  const bool is_slates = _vw->options->was_supplied("slates");
+  return is_ccb_init == is_ccb || is_slates_init == is_slates;
```

Comment:
isn't "slates" inserting "ccb_explore_adf" to args during setup? If so, this check probably will not detect switch between ccb and slates

Reply:
```suggestion
  return is_ccb_init == is_ccb && is_slates_init == is_slates;
```